Pages: add new page about Twitter Impersonation

harding opened the pull request:
In https://github.com/bitcoin-dot-org/bitcoin.org/issues/1842 @midnightmagic requested a page explicitly mentioning the Twitter impersonation accounts are fake.  This PR adds a short statement to that effect, including mentioning Bitcoin Core's official twitter account.

The page is added to the Contact menu:

![2017-10-07-10_18_46_574829865](https://user-images.githubusercontent.com/61096/31308694-014ecddc-ab49-11e7-9b73-9230bc241d31.png)

Note that I reported one of the accounts (*@BcoreProject*) to Twitter about a month ago and it's still there, so I'm not sure we can expect much action from Twitter on this issue.

TheBlueMatt commented:
It is already expliacitly mentioned on https://bitcoincore.org/en/2017/08/18/btc1-misleading-statements/. Hopefully that is enough for whatever Twitter needs to take down the other account.

harding commented:
@TheBlueMatt fair enough.  I'll leave this issue open for another day to see if @midnightmagic has anything to add, since the request was his.

midnightmagic commented:
Mentions about the official twitter and/or webpages are indeed there. I have been actively complaining about the silly impersonation accounts for weeks. It does not appear to be enough to convince twitter. They need a phone number, etc. The procedure is public to close down impersonation, non-parody accounts.

Perhaps they're just not listening to me.

TheBlueMatt commented:
I think we should a) merge this, b) update the text to note that this is a pervasive issue on twitter, not just bitcoin-core-specific. I think this represents an important community service announcement noting that people should generally *not* trust anything they read on twitter is coming from the person it claims to be without very careful analysis

harding commented:
Reopening.  Will make suggested edits presently.

harding commented:
Updated page content:

![2017-10-11-06_48_20_900477964](https://user-images.githubusercontent.com/61096/31436215-3912a6e4-ae50-11e7-9f76-4b0d0cfcdeee.png)

TheBlueMatt commented:
Maybe mention something about not trusting any download links or update announcements specifically, given the security issues there best to be very explicit imo. Otherwise, ACK.

On October 10, 2017 2:54:24 AM EDT, "David A. Harding" <notifications@github.com> wrote:
>Closed #447.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/bitcoin-core/bitcoincore.org/pull/447#event-1285418766

harding commented:
Added notice about downloading software, a link to the download page (/bin for now; will either update this or #452 depending on which gets merged first), and a link to instructions for verifying binaries using GPG.  In addition, I added a paragraph telling people that if they received a security announcement whose origin they weren't sure about, they can just shut down their node to safely take time to investigate.

Also made another edit suggested out of band.

If this looks good, I'll squash before the final merge.

![2017-10-11-14_13_12_374912360](https://user-images.githubusercontent.com/61096/31458715-e12dcb4a-ae8e-11e7-873f-ab697da1ff9e.png)

achow101 commented:
ACK

jnewbery commented:
utACK 10ce3ceaafea6deedcfaebbcfd9602ab028c4962

TheBlueMatt commented:
ACK

harding commented:
Rebased and squashed with no changes from 10ce3ce

midnightmagic commented on _posts/en/pages/2017-01-01-twitter-impersonation.md:
```diff
@@ -0,0 +1,57 @@
+yourself time to investigate.
+
+Although Bitcoin Core does have an official Twitter account,
+[@bitcoincoreorg][], any important announcements about the project will
```
Thank you. This is much appreciated.
